README.md

```diff
@@ -10,208 +10,122 @@
-You can opt-out at any time by running the command:
+## Installation
 
-``` bash
-opt_in_out --opt_out
+[Get your preferred distribution of OpenVINO](https://docs.openvino.ai/2024/get-started/install-openvino.html) or use this command for quick installation:
+
+```sh
+pip install openvino
 ```
 
-More Information is available at https://docs.openvino.ai/latest/openvino_docs_telemetry_information.html.
+Check [system requirements](https://docs.openvino.ai/2024/about-openvino/system-requirements.html) and [supported devices](https://docs.openvino.ai/2024/about-openvino/compatibility-and-support/supported-devices.html) for detailed information.
 
-## Documentation
+## Tutorials and Examples
+
+[OpenVINO Quickstart example](https://docs.openvino.ai/2024/get-started.html) will walk you through the basics of deploying your first model.
+
+Learn how to optimize and deploy popular models with the [OpenVINO Notebooks](https://github.com/openvinotoolkit/openvino_notebooks)📚:
+- [Create an LLM-powered Chatbot using OpenVINO](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/llm-chatbot/llm-chatbot.ipynb)
+- [YOLOv8 Optimization](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/quantizing-model-with-accuracy-control/yolov8-quantization-with-accuracy-control.ipynb)
+- [Text-to-Image Generation](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/controlnet-stable-diffusion/controlnet-stable-diffusion.ipynb)
+
+Here are easy-to-follow code examples demonstrating how to run TensorFlow and PyTorch model inference using OpenVINO:
+
+**TensorFlow Model**
 
-### User documentation
+```python
+import numpy as np
+import openvino as ov
+import tensorflow as tf
 
-The latest documentation for OpenVINO™ Toolkit is available [here](https://docs.openvino.ai/). This documentation contains detailed information about all OpenVINO components and provides all the important information you may need to create an application based on binary OpenVINO distribution or own OpenVINO version without source code modification.
+# load TensorFlow model into memory
+model = tf.keras.applications.MobileNetV2(weights='imagenet')
 
-### Developer documentation
+# convert the model into OpenVINO model
+ov_model = ov.convert_model(model)
 
-[Developer documentation](./docs/dev/index.md) contains information about architectural decisions which are applied inside the OpenVINO components. This documentation has all necessary information which could be needed in order to contribute to OpenVINO.
+# compile the model for CPU device
+core = ov.Core()
+compiled_model = core.compile_model(ov_model, 'CPU')
 
-## Tutorials
+# infer the model on random data
+data = np.random.rand(1, 224, 224, 3)
+output = compiled_model.infer_new_request({0: data})
```

Comment:
can  we use callable instead of `infer_new_request` ?

Reply:
I will address it in separate PR